### PR TITLE
WIP: Update arm64 name to be aarch64

### PR DESCRIPTION
**Pull request by smlambert**
This change needs to be co-ordinated with updating Jenkins job names, and fixes the problem of no aarch64/arm64 builds showing up in the nightlies on the website.

### CODE REVIEW

**Comment by smlambert** [comment]
@sophia-guo - could you review please?

File: pipelines/openjdk10_nightly_pipeline.groovy [hunk 1]
```diff
@@ -1,14 +1,14 @@
 println "building ${JDK_VERSION}"
 
-def buildPlatforms = ['Mac', 'Windows', 'Linux', 'zLinux', 'ppc64le', 'AIX', 'arm64']
+def buildPlatforms = ['Mac', 'Windows', 'Linux', 'zLinux', 'ppc64le', 'AIX', 'aarch64']
```
Comment (sophia-guo) [inline pipelines/openjdk10_nightly_pipeline.groovy]: This value is used as index. Either arm64 or aarch64 is ok.

File: pipelines/openjdk8_nightly_pipeline.groovy [hunk 2]
```diff
@@ -1,13 +1,14 @@
 println "building ${JDK_VERSION}"
 
-def buildPlatforms = ['Mac', 'Windows', 'Linux', 'zLinux', 'ppc64le', 'AIX']
+def buildPlatforms = ['Mac', 'Windows', 'Linux', 'zLinux', 'ppc64le', 'AIX', 'aarch64']
```
Comment (sophia-guo) [inline pipelines/openjdk8_nightly_pipeline.groovy]: I'm not sure if we should add arm64 job to pipeline as it's still unstable? https://ci.adoptopenjdk.net/job/openjdk8_build_arm64_linux/

File: pipelines/openjdk10_nightly_pipeline.groovy (hunk 1 above)
Comment (smlambert) [reply pipelines/openjdk10_nightly_pipeline.groovy]: I think I will change it anyway (even if change not needed, reads better).

File: pipelines/openjdk8_nightly_pipeline.groovy (hunk 2 above)
Comment (smlambert) [reply pipelines/openjdk8_nightly_pipeline.groovy]: in which case, I can remove this change.  I would keep the java8 release pipeline change as its already added there

Comment (gdams) [reply pipelines/openjdk8_nightly_pipeline.groovy]: It should be stable now :)

**Comment by smlambert** [comment]
Ok, I just saw @gdams note about the openjdk8_aarch64 builds being stable now, so will add to the openjdk8 nightly pipeline too.

**Comment by smlambert** [comment]
If I can get one more review, then I will remove the WIP and merge when I rename the builds.

**Comment by smlambert** [comment]
I believe all the build and test jobs are now renamed (thanks to whoever helped!), and also the 3 relevant checksum jobs updated.

**Pull request merged**